### DISCUSSION TITLE
NextDNS using default configuration, but sdns stamp works

### DISCUSSION THREAD

**Discussion by bcookatpcsd** [AdguardTeam/AdGuardHome#3490] [Q&A]
Just wanted to write this before the day got away from me. 

#https://dns.nextdns.io/xyz123
#tls://xyz123.dns.nextdns.io
sdns://biglittleshortstampxyz123

going to my.nextdns.io will show you if you are using their resolver.. 

using the provided doh and dot endpoint it was telling me that I was using the default resolver https://dns.nextdns.io and tls://dns.nextdns.io

It wasn't seeing my profile, until I used the sdns stamp.. then it was recognizing my configuration.. 

AdGuard Home, version v0.107.0-b.8

bootstrap dns didn't seem to matter.. but is presently 9.9.9.10 

setting it to 45.90.28.0 or 45.90.30.0 (nextdns recursor) didn't change either..

**Comment by ameshkov** [comment]
This probably needs to be reported to NextDNS, there's nothing AGH can do wrong here.
